debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/JdbcChangeEventSink.java

```diff
@@ -116,7 +110,7 @@ public void execute(Collection<SinkRecord> records) {
                 }
             }
 
-            if (record.isDelete()) {
+            if (record.isDelete() || record.isTombstone()) {
```

Comment:
Don't you think `isDelete()` should return true when either the message is a Debezium envelope message with `op=d` or when it is a flattend / unwrapped message tombstone? Instead of adding this condition here do it inside `isDelete()`?

Reply:
Perhaps, but my only concern is what happens if we need a situation where we do something specific where we have `op=d` versus a Tombstone; then we've lost that differentiation.

Perhaps the best route in that case would be `isDeleteOrTruncate()` if we really want to hide that detail but be clear we're operating on both.

Reply:
@Naros  but what is wrong with the above code? It's the current implementation of `isDelete()`. So there must be something wrong with that check?!